addons/website_slides/models/slide_slide.py

```diff
@@ -344,6 +356,15 @@ def _post_publication(self):
             )
         return True
 
+    def _get_signed_token(self, partner_id):
+        """ Lazy generate the acces_token and return it signed by the given partner_id
+            :rtype tuple (string, int)
+            :return (signed_token, partner_id)
+        """
+        if not self.access_token:
+            self.write({'access_token': self._default_access_token()})
+        return (self._sign_token(partner_id), partner_id)
```

Comment:
Why returning partner ? It seems returning only the sign result should be sufficient.